scaleway/resource_k8s_cluster_beta.go

```diff
@@ -408,51 +447,42 @@ func resourceScalewayK8SClusterBetaCreate(d *schema.ResourceData, m interface{})
 
 	d.SetId(newRegionalId(region, res.ID))
 
-	err = waitK8SClusterReady(k8sAPI, region, res.ID)
+	err = waitK8SClusterReady(k8sAPI, region, res.ID) // wait for the cluster status to be ready
 	if err != nil {
 		return err
 	}
 
+	if d.Get("default_pool.0.wait_for_pool_ready").(bool) { // wait for the pool status to be ready (if specified)
```

Comment:
Why only wait for `default_pool.0` ? 
What about `default_pool.N` ?

Reply:
There is only 1 default pool :)